### PR TITLE
Improved wording in the mass spring damper example.

### PR DESCRIPTION
**Pull request by moorepants**
@tarzzz 

I updated some minor things here but noticed some bugs too:

1. Even though the integration time is set from 0 to 10 seconds, only 3 seconds plays in the animation.
2. The animation window covers the controls on the right hand side (see attached image).
3. The bob looks like an ellipsoid instead of a sphere (see below).

selection_036

### CODE REVIEW

**Comment by moorepants** [comment]
@jcrist I think this can go in for the examples. I'll check.

**Comment by moorepants** [comment]
Getting this example working again depends on issues #142 and #151.

**Comment by moorepants** [comment]
This should be merged after #194 is merged.

File: pydy/viz/static/js/dyviz/scene.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ DynamicsVisualizer.Scene = Object.extend(DynamicsVisualizer, {
 
             case "Sphere":
                 var geometry = new THREE.SphereGeometry(
-                                               object.radius, 100);
+                                               object.radius, 100, 100);
```
Comment (moorepants) [inline pydy/viz/static/js/dyviz/scene.js]: I'm not sure why this change is there.

**Comment by moorepants** [comment]
I have confirmed that this now works with the master branch.

But it isn't clear to me that the camera is getting properly set. @sahilshekhawat Did you fix the camera's in your previous PR's, or do we need to investigate more?

**Comment by moorepants** [comment]
The camera is definitely not getting rotated correctly in this example.

**Comment by sahilshekhawat** [comment]
I did fixed it and it should work in master.

**Comment by moorepants** [comment]
If you run this example here, you should see that the camera properly oriented the scene as in the first screenshot in this thread. But the orientation is not working, only the camera's location seems to be working.

**Comment by moorepants** [comment]
I'd like to merge this PR. This really only has some rewording of text in the example and a couple of tiny functional changes.

@pydy/pydy-developers Can this get a quick review? I think it should be merged as is.

File: examples/mass_spring_damper/mass_spring_damper.ipynb [hunk 2]
```diff
@@ -287,18 +535,41 @@
...
+      }
+     ],
+     "prompt_number": 15
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "The equations are also available in the form $M\\frac{d}{dt}[q,u]^T=f(\\dot{u}, u, q)$ and we can extract the mass matrix, $M$, and the forcing equations, $f$."
+      "The equations are also available in the form $M\\frac{d}{dt}[q,u]^T=f(q, u)$ and we can extract the mass matrix, $M$, and the forcing equations, $f$."
```
Comment (chrisdembia) [inline examples/mass_spring_damper/mass_spring_damper.ipynb]: I think this should say "forcing functions." Equations have equal signs.

File: examples/mass_spring_damper/mass_spring_damper.ipynb [hunk 3]
```diff
@@ -162,13 +339,14 @@
...
-     "outputs": []
+     "outputs": [],
+     "prompt_number": 8
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "Point $P$ can move downward in the $y$ direction and its velocity is $v$ in the downward direction."
+      "Point $P$ can move downward in the $y$ direction and its velocity is prespecified as $v$ in the downward direction."
```
Comment (chrisdembia) [inline examples/mass_spring_damper/mass_spring_damper.ipynb]: What's the difference between specified and prespecified? Can this just be "specified"?

File: examples/mass_spring_damper/mass_spring_damper.ipynb [hunk 4]
```diff
@@ -426,57 +760,66 @@
...
+     "outputs": [],
+     "prompt_number": 23
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
       "For visualizing the system, we need to create shapes for the objects we wish to visualize, and map each of them \n",
-      "to a VisualizationFrame, which holds the position and orientation of the object."
+      "to a `VisualizationFrame`, which holds the position and orientation of the object. First create a sphere to represent the bob and attach it to the point $P$ and the ceiling reference frame (the sphere does not rotate wrt to the ceiling)."
```
Comment (chrisdembia) [inline examples/mass_spring_damper/mass_spring_damper.ipynb]: Spell out 'with respect to', remove the double 'to'

File: examples/mass_spring_damper/mass_spring_damper.ipynb [hunk 5]
```diff
@@ -426,57 +760,66 @@
...
      "metadata": {},
-     "outputs": []
+     "outputs": [],
+     "prompt_number": 24
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Now create a circular disc that represents the ceiling and fix it to the ceiling reference frame. The circle's default axis is aligned with it's local $y$ axis, so we need to attach it to a rotated ceiling reference frame if we want the circle's axis to align with the $\\hat{c}_x$ unit vector."
```
Comment (chrisdembia) [inline examples/mass_spring_damper/mass_spring_damper.ipynb]: " with it's local" -> "with its local"

File: examples/mass_spring_damper/mass_spring_damper.ipynb [hunk 6]
```diff
@@ -501,11 +845,12 @@
      "cell_type": "code",
      "collapsed": false,
      "input": [
-      "scene.visualization_frames = [bob_vframe, ceil_vframe]\n"
+      "scene.visualization_frames = [bob_vframe, ceiling_vframe]"
```
Comment (chrisdembia) [inline examples/mass_spring_damper/mass_spring_damper.ipynb]: +1

File: examples/mass_spring_damper/mass_spring_damper.ipynb [hunk 7]
```diff
@@ -518,12 +863,15 @@
      "cell_type": "code",
      "collapsed": false,
      "input": [
-      "primary_camera = PerspectiveCamera(ceiling.orientnew('k','Axis', [pi/2, ceiling.z] ), O.locatenew('a',100*ceil_reference_frame.z))\n",
+      "camera_frame = ceiling.orientnew('Camera Frame','Axis', [pi / 2, ceiling.z])\n",
+      "camera_point = O.locatenew('Camera Location', 100 * camera_frame.z)\n",
+      "primary_camera = PerspectiveCamera(camera_frame, camera_point)\n",
```
Comment (chrisdembia) [inline examples/mass_spring_damper/mass_spring_damper.ipynb]: +1

**Comment by chrisdembia** [comment]
I made some comments. My review is partial. I don't know enough to say whether or not this should be merged.

**Comment by moorepants** [comment]
Thanks. I'll change them.

Why partial?

I'll merge after these fixes. This is too minor to let it continue lingering.

**Comment by moorepants** [comment]
Ok, this is going in. Thanks @chrisdembia.
